ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-4.14.yaml

```diff
@@ -78,6 +78,28 @@ tests:
         MIRROR_IMAGES=true
```

Comment:
the changes should be in master branch, otherwise they will not be triggered on master PRs (almost sure)

Reply:
Adding 4.14 to the tests would prevent testing OVN and 4.15 in those scenarios, but I will duplicate them in master with 4.14 specified

Reply:
Seems reasonable until SDN will be applicable for 4.15 no ?